File: iTSoft.CRM.Domain/Models/ViewModel/RequestSelectListModel.cs
using iTSoft.CRM.Data.Entity;
using iTSoft.CRM.Data.Entity.Master;
using System;
using System.Collections.Generic;
using System.Text;

namespace iTSoft.CRM.Domain.Models.ViewModel
{
   public class RequestSelectListModel
    {
        public List<LeadStatusMaster> LeadStatuses { get; set; }
        public List<ListModel> Stages { get; set; }
        public List<ListModel> Sources { get; set; }
        public List<ListModel> ClientBehaviour { get; set; }
        public List<ListModel> Departments { get; set; }
        public List<ListModel> OrganizationTypes { get; set; }

    }
}

File: iTSoft.CRM.UI/src/app/_models/requestselectlistmodel.ts
import { StatusMaster } from '../masters/status/status.model';
import { ListModel } from './listmodel';

export class RequestSelectListModel
{
    LeadStatuses: StatusMaster[];
    Stages: ListModel[];
    Sources: ListModel[];
    ClientBehaviour: ListModel[];
    Advisors : ListModel[];
    Departments : ListModel[];
    OrganizationTypes : ListModel[];
    Services: ListModel[];
    constructor()
    {
        this.LeadStatuses=[];
        this.Stages=[];
        this.Sources=[];
        this.ClientBehaviour=[];
        this.Advisors = [];
        this.Departments = [];
        this.OrganizationTypes = [];
        this.Services = [];
    }
}